packages/electrode-archetype-react-app-dev/config/babel/babelrc-server

```diff
@@ -0,0 +1,5 @@
+{
+  "presets": [
+    "es2015-node"
```

Comment:
Will excluding React transforms here prevent us from using React as our template language on the back end? I know most of our examples don't do that, but I think that is a pretty common thing to do.